packages/fp-behandling/src/fakta/components/person/panelBody/arbeidsforhold/PersonArbeidsforholdPanel.jsx

```diff
@@ -101,8 +103,8 @@ const leggTilValuesForRendering = arbeidsforholdList => arbeidsforholdList.map(a
   ...arbeidsforhold,
   originalFomDato: arbeidsforhold.fomDato,
   brukUendretArbeidsforhold: sjekkBrukUendretArbeidsforhold(arbeidsforhold),
-  fortsettUtenImAktivtArbeidsforhold: sjekkFortsettUtenImAktivtArbeidsforhold(arbeidsforhold),
-  overstyrtTom: arbeidsforhold.tomDato,
+  aktivtArbeidsforholdFortsettBehandlingUtenIM: sjekkAktivtArbeidsforholdFortsettBehandlingUtenIM(arbeidsforhold),
+  overstyrtTom: arbeidsforhold.brukMedJustertPeriode ? arbeidsforhold.tomDato : undefined,
 }));
 
 // -------------------------------------------------------------------------------------------------------------
```

Comment:
Kommentarer for komponenter skal vere på formen
```suggestion
/**
 *  PersonArbeidsforholdPanel
 *
 * Forklaring av kva komponenten gjer og spesielle hensyn som må tas.
 *
 */
```